mirrormanager2/app.py

```diff
@@ -1373,9 +1373,9 @@ def set_session():
 
 
 def statistics_file_name(date, cat, ext):
-    year = date.year
-    month = date.month
-    day = date.day
+    year = date.strftime('%Y')
+    month = date.strftime('%m')
+    day = date.strftime('%d')
```

Comment:
Oh, the files have zero padded dates like 2015-12-05 and .day returns an integer. So that breaks. The output file is generated with strptime('%Y-%m-%d') and is thus zero padded.